lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -121,6 +121,29 @@ static unsigned getPrintedIntWidth(unsigned value) {
   return stream.str().size();
 }
 
+/// Calculated the printed width of the array dims of a type.
+static size_t getPrintedTypeDimsWidth(Type type, Operation *op) {
+  int bitWidth;
+  size_t textWidth = 0;
+  if (auto array = type.dyn_cast<rtl::ArrayType>()) {
+    bitWidth = array.getSize();
```

Comment:
This really isn't a bitwidth in the array.  Maybe a better name would be rangeSize?